src/apirequest.cpp

```diff
@@ -23,17 +36,46 @@ void ApiRequest::makeApiGetRequest(const QString &method, const QUrlQuery &q) {
     url.setQuery(query.query());
     QNetworkRequest request(url);
     request.setRawHeader("Authorization", "OAuth "+accessToken.toUtf8());
-    connect(_manager, &QNetworkAccessManager::finished, this, &ApiRequest::dataReady, Qt::QueuedConnection);
+    request.setRawHeader("Accept", "*/*");
+    request.setRawHeader("X-Yandex-Music-Content-Type", "adult");
+    request.setHeader(QNetworkRequest::UserAgentHeader, "Maple/519 (iPhone; iOS 12.2; Scale/2.00)");
```

Comment:
Really needed ?

Reply:
Yes, these headers are used in such requests in the official app.